Add Offers to Sawbuck data submission

delventhalz opened the pull request:
Also includes an implementation of a simple reference language which can be used to fetch data from previously submitted resources. In this case it is used to get ids from Holdings.

Commit:
Add Offers to Sawbuck data submission

Also includes an implimentation of a simple reference language
which can be used to fetch data from previously submitted resources.
In this case it is used to get ids from Holdings.

Signed-off-by: Zac Delventhal <zac@bitwise.io>

boydjohnson reviewed (approved):
Ran it locally. It didn't come in with this PR, but oddity between `-u http://localhost:8040` and `-u localhost:8040`. The second works but the first does not. I haven't been able to figure it out, but haven't put much time into it.

Commit:
Remove source type rule from seeded data

This rule is unneccessary to the structure of the assets. Since
they are not transferable, they will only ever be traded in
offers set up by the asset owner.

Signed-off-by: Zac Delventhal <zac@bitwise.io>

Commit:
Fix admin script RegEx use

Contrary to what you might expect, re.match only searchs the start of
the string.

Signed-off-by: Zac Delventhal <zac@bitwise.io>

delventhalz commented:
@boydjohnson Python is bad. I added a fix.
